### DISCUSSION TITLE
Code, fix warnings : help testing appreciated

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#91] [Q&A]
To face more easily the compatibility with the new version 8.3 of PHP, and the later versions to come, I have started a branch where I have changed the error handler to show all warnings (& errors of course).  I fixed already a lot of code to prevent warnings.  The quantity of changes is somehow important, I would appreciate getting help to test and find possible problems I would have created.

The branch is "code_no_warnings".

I thank already any help I will get !

**Comment by devinemke** [comment]
The **code_no_warnings** branch seems to have no effect on issue #88 that I created a few weeks ago. I gather there are no changes to `compile.php` in that branch.

**Reply by LionelLaffineur** [reply]
Dear @devinemke, thanks for your testing.  I have added an extra commit where I have fixed warnings from compile.php.
